upgrade/1.4.2/README.md

```diff
@@ -174,6 +174,53 @@ This step will install the `smart-mon` rpm on storage nodes, and reconfigure the
    /usr/share/doc/csm/scripts/operations/ceph/enable-smart-mon-storage-nodes.sh
    ```
 
+## Configure NCN nodes without restart
+
+This step will create an imperative CFS session that can be used to configure booted NCN nodes with updated sysctl values.
+
+1. (`ncn-m001#`) Create a new CFS Configuration Entry for the 1.4.2 release
+
+```bash
+VCS_USER=$(kubectl get secret -n services vcs-user-credentials --template={{.data.vcs_username}} | base64 --decode)
```

Comment:
Ideally we should get the commit ID from the Cray Product Catalog, IMO

Reply:
Agreed and done.